api/src/main/java/io/hyperfoil/function/SerializableToObjectFunction.java

```diff
@@ -0,0 +1,6 @@
+package io.hyperfoil.function;
+
+import java.io.Serializable;
+
+public interface SerializableToObjectFunction<T> extends Serializable {
```

Comment:
This has no method and is not a function.

Reply:
Removed `SerializableToObjectFunction `